docs/docs/cmd/login.md

```diff
@@ -93,3 +99,15 @@ Log in to Microsoft 365 using a user-assigned managed identity. Client id or pri
 ```sh
 m365 login --authType identity --userName ac9fbed5-804c-4362-a369-21a4ec51109e
 ```
+
+Log in to Microsoft 365 using your own multitenant Azure AD application
+
+```sh
+m365 login --appId 31359c7f-bd7e-475c-86db-fdb8c937548c
+```
+
+Log in to Microsoft 365 using your own Azure AD application that's restricted only to allow accounts from the specific tenant
+
+```sh
+m365 login --appId 31359c7f-bd7e-475c-86db-fdb8c937548c --tenant 31359c7f-bd7e-475c-86db-fdb8c937548a
```

Comment:
We should use different ids in the example here I think, just for completeness as you would never have an appId and tenantId being the same.

Reply:
They're not, see the `c` vs `a` at the end, but I get it's too subtle 😄